pkg/quota/admission/clusterresourcequota/accessor.go

```diff
@@ -147,10 +148,14 @@ func (e *clusterQuotaAccessor) waitForReadyClusterQuotaNames(namespaceName strin
 	err := utilwait.PollImmediate(100*time.Millisecond, 8*time.Second, func() (done bool, err error) {
 		var namespaceSelectionFields clusterquotamapping.SelectionFields
 		clusterQuotaNames, namespaceSelectionFields = e.clusterQuotaMapper.GetClusterQuotasFor(namespaceName)
-		namespace, err := e.namespaceLister.Get(namespaceName)
+		obj, ok, err := e.namespaceLister.Get(&kapi.Namespace{ObjectMeta: kapi.ObjectMeta{Name: namespaceName}})
```

Comment:
This changed semantics.  Restore the old behavior (fail on namespace get failure) and we'll try to fix the real problem in https://trello.com/c/Issi3JwT/11-controller-standardize-listers-they-should-have-consistent-label-field-selector-behavior-and-should-report-normal-api-errors-loo